Dockerfile-versioned-source

```diff
@@ -0,0 +1,72 @@
+FROM golang:1.19-alpine
```

Comment:
What is the purposes of this dockerfile? And why is it a different version than the others?

Reply:
I created it, it's to perform stateful smoke tests using current source instead of a version, didn't realize Golang version was different

Reply:
https://github.com/zeta-chain/node/issues/1433